update_jets.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+set -e
+
+
+if [ -z "$1" ]; then
+  echo "\$1 parameter must be the simplicity directory"
+  echo "\$2 parameter (optional) can be the revision to check out"
+  exit 1
+fi
+
+SRC_DIR=$1
+REV=$2
+ORIGDIR=$(pwd)
+
+cd "$SRC_DIR" || exit 1
+if [ -n "$REV" ]; then
+    git checkout "$REV"
+fi
+HEAD1=$(git rev-parse HEAD)
+cd $ORIGDIR
```

Comment:
In 8a8be85d857af7a55d2c8076f96eaee5957c524b:

should add quotes around `$ORIGDIR`

Reply:
There are several other cases of missing quotes -- can you run `shellcheck` locally on this file to get a full list? I'm happy to comment them all but it might get annoying :)